_posts/newsletters/2023-01-24-newsletter.md

```diff
@@ -0,0 +1,215 @@
+     * <a name="sustain_podcast"></a>[Sustaining Open Source Ecosystems Podcasts](https://podcast.sustainoss.org)
+* The organizers of RSECon2022 have released a playlist with 38 videos so far 
+    * <a name="https://www.youtube.com/playlist?list=PL27mQJy8eDHk---gHCapHwyVaWIkyORYj"></a>[RSECon2022 Playlist](https://www.youtube.com/playlist?list=PL27mQJy8eDHk---gHCapHwyVaWIkyORYj)
+* The Diátaxis framework aims to solve the problem of [structure in technical documentation](https://diataxis.fr)
+* [11 Most In-Demand Programming Languages](https://bootcamp.berkeley.edu/blog/most-in-demand-programming-languages/)
+* [Inclusive Scientific Meetings](https://500womenscientists.org/inclusive-scientific-meetings)
+* [Build Inclusive Products](https://makeitfable.com/)
+
+<a name="outreach"></a>
+# **US-RSE Outreach**
+
+## RSEs in eScience 2022 Workshop Report
+
+If you were unable to come to Salt Lake City, UT, for the inaugural [RSEs in eScience Workshop](https://us-rse.org/rse-escience-2022), the Outreach Working Group posted a report on the talks and breakout discussion. [See for yourself](https://us-rse.org/2023-01-17-escience-2022-report-out/)!
+
+
+The Outreach Working Group is looking for volunteers to help with organizing workshops, birds of a feather (BoF), symposiums, etc. Roles span from organizer to reviewer to speaker and more.
+
+If you are interested in getting involved in a US-RSE hosted event, please reach out to the working group via the [#wg-outreach](https://app.slack.com/client/T8ZT4PJSW/CNR6E0N75) slack channel or [wg-outreach@us-rse.org](mailto:wg-outreach@us-rse.org).
+
+<a name="financials"></a>
+# **See how we are doing financially**
+
+We currently organize US-RSE finances transparently via the [Open Collective](https://opencollective.com/usrse) project. You can see all budgets, expenses, and contributions on [this site](https://opencollective.com/usrse). Anyone can go to the site and see the current accounts, income and expenses and ….. (if the mood takes you and you are able) **[make a tax-deductible gift to US-RSE](https://opencollective.com/usrse#category-CONTRIBUTE)**!
+
+<a name="involved"></a>
+# **Get Involved**
+
+There are lots of ways to get involved with the US-RSE community. Of course, you can [join us on Slack](https://us-rse.org/join). We’re also looking for ideas and help in many other places. See our [list of projects](https://docs.google.com/document/d/1jjVD0WkeeWZJI6yqSKyMdIjtClzolsxv75RkpLju17I/edit?usp=sharing) and let us know how you’d like to help. Help with things like the Website Committee, Social Media, Community Engagement, Fundraising, and Newsletter are all needed and welcome.
+
+As always, we thank everyone who is already contributing for their help in shaping and growing the organization. In 2021 we added nearly 300 new members. There are still many more RSEs and friends of RSEs we would like to help connect and support. If you are interested in helping in any way please feel free to join in.
```

Comment:
why are we talking about how many new members we added in 2021? Let's talk about the larger number we added in 2022 instead!

Reply:
Great point! We added 605 in 2022.